examples/advanced_diffusion_training/train_dreambooth_lora_sdxl_advanced.py

```diff
@@ -1995,13 +1945,13 @@ def compute_text_embeddings(prompt, text_encoders, tokenizers):
     if accelerator.is_main_process:
         unet = accelerator.unwrap_model(unet)
         unet = unet.to(torch.float32)
-        unet_lora_layers = unet_lora_state_dict(unet)
+        unet_lora_layers = get_peft_model_state_dict(unet)
 
         if args.train_text_encoder:
             text_encoder_one = accelerator.unwrap_model(text_encoder_one)
-            text_encoder_lora_layers = text_encoder_lora_state_dict(text_encoder_one.to(torch.float32))
+            text_encoder_lora_layers = get_peft_model_state_dict(text_encoder_one.to(torch.float32))
```

Comment:
If you train extra parameters that you keep unfrozen for text encoder, you need to add them in `modules_to_save=[xxx]` when defining the lora config for the text encoder

Reply:
Got it! But I'm not training extra parameters with that operation. With `args.train_text_encoder` we're doing regular text encoder training. 

However, with `args.train_text_encoder_ti` (that is mutually exclusive with `args.train_text_encoder`) then the goal is to freeze all but the `token_embedding` of the text encoder and train the text embeddings for the new tokens introduced in the model.

This is where is taking place - and was working prior to PEFT: https://github.com/huggingface/diffusers/pull/6294/files#diff-24abe8b0339a563b68e03c979ee9e498ab7c49f3fd749ffb784156f4e2d54d90R1249

Reply:
So, only thing that is outside of the `peft` paradigm currently is what's happening when `args.train_text_encoder_ti` is True, yeah?

Reply:
Yes, which makes sense because it is not training an adapter per se

Reply:
Thanks @apolinario for explaining ! makes sense